### DISCUSSION TITLE
Setting environment variables

### DISCUSSION THREAD

**Discussion by skarekrow** [ghostty-org/ghostty#3776] [Q&A] [answered]
Hello!

I've been poking around trying to figure out how to set `XDG_CONFIG_HOME` in ghostty so that my configuration for nushell is honored when it's invoked as the command. 

Is there support for this in the configuration? I haven't been able to find any on the website.

Thanks!

**Comment by skarekrow** [comment, marked as answer]
Well a little ugly atm, but this works in the config :D

```
command = env XDG_CONFIG_HOME="/some/path/.config" PATH="/opt/homebrew/bin:~/.atuin/bin:$PATH" EDITOR="nvim" VISUAL="nvim" /opt/homebrew/bin/nu -l
```

**Reply by rhodes-b** [reply]
`command` is the way to inject a command at startup so this is correct
